reana_client/cli/workflow.py

```diff
@@ -513,6 +513,53 @@ def workflow_validate(ctx, file):
             err=True)
 
 
+@click.command(
+    'stop',
+    help='Stop a running workflow')
+@click.option(
+    '--force',
+    'force_stop',
+    is_flag=True,
+    default=False,
+    help='Stop a workflow without waiting for jobs to finish.')
+@click.option(
+    '-w',
+    '--workflow',
+    default=os.environ.get('REANA_WORKON', None),
+    callback=workflow_uuid_or_name,
+    help='Name and run number to be stopped. '
+         'Overrides value of REANA_WORKON.')
+@add_access_token_options
+@click.pass_context
+@with_api_client
+def workflow_stop(ctx, workflow, force_stop, access_token):
+    """Stop given workflow."""
+    if not force_stop:
+        click.secho('Graceful stop not implement yet. If you really want to '
```

Comment:
Works fine like this, we could consider showing a confirmation message and having `--force` set by default